Dockerfile

```diff
@@ -0,0 +1,35 @@
+# FROM node:8-alpine also works here for a smaller image
+FROM node:8-alpine
```

Comment:
How do you plan to "vendor" the generated Docker images for self-contained delivery? Do you plan to publish it to Docker Hub for easy external consumption as well?

Reply:
> How do you plan to "vendor" the generated Docker images for self-contained delivery? Do you plan to publish it to Docker Hub for easy external consumption as well?

May be host the image on Nexus, because Docker Hub will still need internet access

Reply:
@abh1kg Currently, we plan to publish it on external docker hub. self-contained delivery is still not thought of. May be use of private registry or Nexis as @akskap said could be a solution.

Reply:
As of now, Nexus is not available for Docker images. Artifactory is one option of course. But, it makes sense to include in Docker Hub as well for external users who want to try out the new framework and get started quickly.
In this case, the deployment model should be "configurable" with the repository itself being a parameter